NAMESPACE

```diff
@@ -1,5 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
+export(extract_hiv_population)
```

Comment:
I don't love these names - before we make the package we should consider the strategy for designing the API

Reply:
Agreed there not amazing, I wonder if you have any good suggestions? At least for these two new functions. If they are going to be used by others it would be nice to get something good now as to not change them in the future. 

For this name I tried to use their language i.e. the request was "we need to extract PLHIV and population"